DEVDOCS-5033: [new] B2B getting started + GraphQL Storefront API auth narrative (#288)
bigcommerce/docs: docs/b2b-edition/about/api-status-codes.mdx

| **406** | **Not Acceptable** | When the client specifies a response content type in the `Accept` header that is not supported. |
| **409** | **Conflict** | A change requested by the client is being rejected because it did not meet a condition imposed by the server. The exact reasons for this response vary from one resource to another. For example, attempting to delete a category whose deletion would orphan products. Additional information about the conflict, and about how to resolve it, may be available in the response's `details` section. |
| **413** | **Request Entity Too Large** | When the client requests too many objects. For example, the `limit` parameter exceeded the maximum. |
| **415** | **Unsupported Media Type** | There are issues with the `Content-Type` header.
| **422** | **Missing or Invalid Data** | The request could not be processed, either because it omitted required fields or because it contained invalid data. See the response for more details. |

> | **429** | **Too Many Requests** | When an OAuth client exceeds the [rate limit](/docs/api-docs/best_practices#best-practices_rate-limits) for API requests to a store. |

Author: ```suggestion
| **429** | **Too Many Requests** | When an OAuth client exceeds the [rate limit](/docs/start/best-practices#api-rate-limits) for API requests to a store. |
```

## 5xx Server Error

5xx codes are returned for requests that could not be processed because there was an internal error with the API or server.

| Code | Text | Purpose |
|-|-|-|
| **500** | **Internal Server Error** | When an error has occurred within the API. |
| **501** | **Not Implemented** | When a request method is sent that is not supported by the API, such as `TRACE` or `PATCH`. |
| **503** | **Service Unavailable** | When the store is “Down for Maintenance”, being upgraded to a new version, or is suspended due to administrative action or a billing issue.|
| **507** | **Insufficient Storage** | When the store has reached a limitation for the resource. |

## Troubleshooting

| Code | Common Causes | Solutions |
|-|-|-|
|**204**, **301**, and **302**| Redirects | Try the `www` or non-`www` version of the URL. |
|**400**|Invalid syntax, required data missing, `Content-Type` header missing|Double-check request body for syntax errors and missing data; check the `Content-Type` header. |
|**401** |API credentials are missing or invalid.|Double-check the `authToken`. Send cURL request with the same credentials to rule out app or config issues. |

> |**403**| API lacks required OAuth scopes, a store owner account changed, operations resulting from API request exceed a platform limit, or URL requested is incorrect.|Double-check OAuth Scopes. Check the URL. Are the endpoint and `authToken` correct? |

Author: ```suggestion
|**403**| API account lacks required OAuth scopes, the store owner account changed, the requests exceeded an internal platform limit, or URL requested is incorrect.|Double-check OAuth Scopes. Check the URL. Are the endpoint and `authToken` correct? |
```
